packages/react/src/components/hero/Hero.stories.tsx

```diff
@@ -667,3 +667,47 @@ AllHeroes.argTypes = {
   ...getDisabledControl('centeredContent'),
   ...getDisabledControl('variant'),
 };
+
+// a loremIpsum function creating at least 300 letters long string with spaces
+const loremIpsum = () => {
+  return 'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec euismod, quam sed aliquet faucibus, nisl risus condimentum nisl, quis aliquam ligula mauris nec enim. Donec euismod, quam sed aliquet faucibus, nisl risus condimentum nisl, quis aliquam ligula mauris nec enim. Donec euismod, quam sed aliquet faucibus, nisl risus condimentum nisl, quis aliquam ligula mauris nec enim.';
+};
+
+export const AllHeroesWithArrowIcon = () => {
+  const DummyContent = () => {
+    return (
+      <div style={{ minHeight: '200px', padding: '20px' }}>
+        <h1>{loremIpsum()}</h1>
```

Comment:
This should not be h1, because Hero has h1 already.

Reply:
And it is awfully long h1 😄

Reply:
Ok I'll revise this a bit :D